heat/core/statistics.py

```diff
@@ -515,6 +516,67 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None):
     return c
 
 
+def histc(input, bins: int = 100, min: int = 0, max: int = 0, out=None):
```

Comment:
can you add an alias for `histogram` to allow for numpy API compatibility? (also i think its easier for users)